css-scroll-snap-2/Overview.bs

```diff
@@ -279,13 +279,35 @@ Snap Events {#snap-events}
 				<th>Event handler event type
 		<tbody>
 			<tr>
-				<th><dfn event>snapChanged</dfn>
+				<th><dfn event>snapchanged</dfn>
 				<td>{{scroll!!event}}
 			<tr>
-				<th><dfn event>snapChanging</dfn>
+				<th><dfn event>snapchanging</dfn>
 				<td>{{scroll!!event}}
```

Comment:
Should we expand the table to include the targets and descriptions for these events similar to https://drafts.csswg.org/cssom-view-1/#eventdef-document-scroll or perhaps like https://w3c.github.io/uievents/#events-wheel-types ?

I think event handler event type is meant to be the type of the event which I believe should be the new type you've defined below.

Reply:
Good suggestion. The table is now expanded to include "Interface", "Targets", & "Description" columns. The Targets for snapchanged is "scroll containers" and not snap container only because of the one case where snapchanged happens because of a style change from non-none scroll-snap-type to none, where technically, the event target is no longer a snap container. (Happy to change it if making this distinction isn't helpful).